app/lib/github.rb

```diff
@@ -1,10 +1,44 @@
 # frozen_string_literal: true
 
 class Github
-  def self.client
+  def self.bot
     @client ||= Octokit::Client.new(
       auto_paginate: true,
       access_token: Rails.application.credentials.github[Rails.env.to_sym][:bot_token]
     )
   end
+  
+  def self.client
+    #TODO: Do not generate jwt and app token per request!
+    Octokit::Client.new(
+      auto_paginate: true,
+      access_token: app_token
+    )
+  end
+
+  def self.jwt
+    private_pem = Rails.application.credentials.github[Rails.env.to_sym][:private_key]
+    private_key = OpenSSL::PKey::RSA.new(private_pem)
+    
+    # Generate the JWT
+    payload = {
+      # issued at time
+      iat: Time.now.to_i,
+      # JWT expiration time (10 minute maximum)
+      exp: Time.now.to_i + (10 * 60),
+      # GitHub App's identifier
+      iss: Rails.application.credentials.github[Rails.env.to_sym][:app_id]
+    }
+    
+    jwt = JWT.encode(payload, private_key, "RS256")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless assignment to variable - `jwt`.](https://app.codacy.com/app/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=4479612)